src/main/resources/db/changelog/changeset/01-create-processing-table.yaml

```diff
@@ -0,0 +1,55 @@
+databaseChangeLog:
+  - changeSet:
+      id: 01-create-processing-table
+      author: freesky15
+      changes:
+        - createTable:
+            tableName: processingresults
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: arrivaltasktime
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: taskid
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: solve
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: userid
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: metrics
+                  type: jsonb
+                  constraints:
+                    nullable: true
+              - column:
+                  name: state
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: departureresulttime
```

Comment:
What does the field means: departure to where?

Reply:
This field means time of sending to the result service(the kafka topic) or back to competition service(if the result is false)

Reply:
I don't think it's required to handle, especially if you have debezium which will send the event to kafka itself. If you want to handle history of event delivery to kafka, it's better to look at the debezium events table to understand what happened.

Reply:
Ok, I see

Reply:
Done!